metrics/config.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Knative Authors
+Copyright 2019 The Knative Authors
```

Comment:
We haven't been doing changes to copyright notices for existing files. Could you revert these, new files using 2019 is fine of course?

Reply:
Done. Thanks.